frontend/components/Radio.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+
+    <label class="control-container">Hello
+        <input type="radio" name="asdf" value="123" checked>
```

Comment:
Can we set up props for these in this PR, since we've already started setting up props for other elements?

Reply:
Yup, my bad, forgot!